p2p/src/swarm/tests/ban.rs

```diff
@@ -51,11 +54,26 @@ async fn ban_connected_peer() {
 }
 
 #[tokio::test]
-async fn banned_peer_attempts_to_connect() {
+async fn ban_connected_peer_libp2p() {
+    ban_connected_peer::<Libp2pService>(make_libp2p_addr(), make_libp2p_addr()).await;
+}
+
+#[tokio::test]
+async fn ban_connected_peer_mock() {
+    // TODO: implement `ban_peer()`
```

Comment:
Should we create an issue in such cases? In order to not forget about it.

Reply:
good idea, #326 tracks this issue